src/components/organelle-wrapper.tsx

```diff
@@ -83,21 +85,23 @@ class OrganelleWrapper extends React.Component<OrganelleWrapperProps, OrganelleW
       this.completeLoad();
     });
 
-    autorun(() => {
-      if (rootStore.mode === Mode.Normal) {
-        this.setState({hoveredOrganelle: null});
+    this.handler = autorun(() => {
+      if (this.model) {
+        const newModelProperties = this.props.organism.modelProperties;
```

Comment:
I don't think this is *quite* right. Like you mentioned, MobX behaves weirdly with nested properties. I found more info on that here:

https://mobx.js.org/best/pitfalls.html#-object-somenewprop-value-is-not-picked-up

I tried setting some break points to figure out when this handler was called and I couldn't understand the behavior. If I removed the `rootStore.mode` check, this handler was never called at all, despite the updates to the `modelProperties`. Depending on the other property to call these updates seemed off, so I spent (probably too much time) getting it working in a way that looks better to me.